tasks/release.py

```diff
@@ -1248,13 +1250,16 @@ def create_rc(ctx, major_versions="6,7", patch_version=False, upstream="origin")
             code=1,
         )
 
-    create_pr(
+    pr_url = create_pr(
         f"[release] Update release.json and Go modules for {versions_string}",
         current_branch,
         update_branch,
         new_final_version,
     )
 
+    # Step 4 - If slack workflow webhook is provided, send a slack message
+    ctx.run(f"curl -X POST -H 'Content-type: application/json' --data '{{\"pr_url\":\"{pr_url}\"}}' {slack_webhook}")
```

Comment:
Do you have an example of the slack message that will be sent?

Reply:
That will depend on the webhook configuration. I created slack workflow that consumes this and produces message like: "New Agent RC PR has been created - {pr_url}".
I thought about making this generic and maybe pass the whole message in curl, but on the other hand - we don't really need to use this when calling task manually, so I went this way. I am open for suggestions though if you think it could be handled better.